src/assets/i18n/nb.json

```diff
@@ -161,6 +161,7 @@
     "MAP_LEGENDS": "Tegnforklaring for kart",
     "MAP_SETTINGS": "Kartvalg",
     "MAP_SETTINGS_FOR": "Kartvalg for",
+    "NO_EMAIL_APP": "Epost kan ikke sendes. Hvis du er Iphone bruker du må ha Mail app til å sende eposter direkte fra Regobs appen.",
```

Comment:
Kan vi gjøre det lettere for bruker å forstå hva som er feil og hvordan de kan fikse det?
Det mangler også en forklaring på hva som er feil (årsak) for Android-brukere.
Her er nytt forslag:
_Epost kan ikke sendes, fordi vi ikke får kontakt med epost-appen på telefonen. På iPhone må du ha appen "Mail"._
(har ikke oversatt til engelsk, det gjør du sikkert mye bedre enn meg:)